runtime/vm/jvminit.c

```diff
@@ -1822,13 +1822,6 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 					j9port_control(J9PORT_CTLDATA_VMEM_ADVISE_HUGEPAGE, 1);
 				} else if (argIndex > argIndex2){
 					j9port_control(J9PORT_CTLDATA_VMEM_ADVISE_HUGEPAGE, 0);
```

Comment:
It would be good to leave a comment in the code that the default value is set in OMR (and provide a pointer to that code) as well as mention that the default right now is only enabled for linux madvise by default.

Reply:
Updated comments as suggested